packages/inspect-utils/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "@effection/atom": "2.0.2",
-    "@effection/dispatch": "2.0.1",
+    "@effection/dispatch": "2.0.2",
```

Comment:
Is there a reason to bump this?

Reply:
For this PR, no. But it fixes a problem that `inspect-utils` uses old version, not the same from the monorepo. I can move that change in separate PR